### ISSUE TITLE
Collapse component console warning

### ISSUE THREAD

**Issue by JoseDuSV** [kiwicom/orbit-components#1375]
Missing `aria-label` on ButtonLink inside [Collapse component](https://github.com/kiwicom/orbit-components/blob/2b6196a215bd16f20d9cee1e57bd1081225e96d7/src/Collapse/index.js#L136) is throwing [console error](https://monosnap.com/file/SqsuU8IgMR1GtEpkSmSZr3a26LLtIj).

## Possible Solution
Define `aria-label` prop.
